### PR TITLE
feat(pci-billing): add layout for tabs routing

### PR DESCRIPTION
**Pull request by kqesar**
ref: DTCORE-2794

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | DTCORE-2794
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  }, [location.pathname]);
+
+  return (
+    <>
+      <Headers
+        title={t('cpbc_billing_control')}
+        headerButton={<PciGuidesHeader category="instances" />}
+      />
+
+      <OsdsBreadcrumb
```
Comment (seven-amid) [inline packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx:66]: The breadcrumb should be displayed before the header i think !

**Review by seven-amid — APPROVED** [review]

File: packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx (hunk 1 above)
Comment (kqesar) [reply packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx:61]: updated

File: packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  Outlet,
+  useHref,
+  useLocation,
+  useResolvedPath,
+} from 'react-router-dom';
+import { sub } from 'date-fns';
+import { ROUTE_PATHS } from '@/routes';
+import TabsPanel from '@/components/TabsPanel.component';
+
+export default function Pools() {
```
Comment (frenautvh) [inline packages/manager/apps/pci-billing/src/pages/billing/Billing.page.tsx:21]:
```suggestion
export default function BillingPage() {
```

File: packages/manager/apps/pci-billing/src/components/TabsPanel.component.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+export type TabsProps = {
+  tabs: TabItemProps[];
+};
+
+export default function TabsPanel({ tabs }: Readonly<TabsProps>) {
+  const [activePanel, setActivePanel] = useState('');
+  const location = useLocation();
+  const { clearNotifications } = useNotifications();
+  const { tracking } = useContext(ShellContext)?.shell || {};
```
Comment (sidlynx) [inline packages/manager/apps/pci-billing/src/components/TabsPanel.component.tsx:27]:
```suggestion
  import { useTracking } from '@ovh-ux/manager-react-shell-client';
```

This way you can mock it while uni testing without having to mock useContext(i got the case)

**Review by sidlynx — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/pci-billing/src/components/TabsPanel.component.tsx (hunk 3 above)
Comment (kqesar) [reply packages/manager/apps/pci-billing/src/components/TabsPanel.component.tsx:27]: it's deprecated https://github.com/ovh/manager/blob/master/packages/manager/core/shell-client/src/hooks/useTracking.ts#L5

**Pull request merged**